libs/ledger-live-common/src/network.ts

```diff
@@ -46,7 +46,14 @@ export const errorInterceptor = (error: AxiosError<any>): AxiosError<any> => {
 };
 
 axios.interceptors.response.use(undefined, errorInterceptor);
-
+// not react native
```

Comment:
yes, it is global instead of btc family only. 
I don't find a way to isolate it. but I don't think there is any side effects. I wait for @gre for the review.
Comments added

Reply:
Also, no need for `keepAlive: true` when we are in ReactNative environment?

Reply:
In fact, the ```const https = require("https");``` doesn't work in react-native.
the httpsAgent is only a module for nodejs

In react-native, we actually don't need to manually set "keep-alive". React Native's internal network handling mechanism will automatically manage the reuse of connections. 
In React Native, we do not need (and cannot) set https.Agent({ keepAlive: true }) as we would in Node.js. This is because the underlying network handling in React Native is not performed by Node.js's http or https modules, but by browser-level network APIs (such as XMLHttpRequest) or native network APIs, which have already dealt with connection reuse issues.